echo-int-test-vars.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
Did you mean to commit this?

Reply:
yes, it generates the run config string you can use to run integration tests in the ide